src/App.tsx

```diff
@@ -0,0 +1,124 @@
+import { useRef, useState } from "react";
+import { NumberButton } from "./components/NumberButton";
+import { OperationButton } from "./components/OperationButton";
+import { CALCULATOR_NUMBERS, OPERATION_REGEX } from "./constants";
+import "./css/index.css";
+import { EOPERATIONS } from "./enums";
+import { calculateByOperator } from "./helper";
+
+function App() {
+  /**화면에 표시되는 수식들 */
+  const [inputtedTotalValue, setInputtedTotalValue] = useState<string>("0");
+  /**사용자가 현재 입력하고 있는 숫자 */
+  const currNumber = useRef<string>();
+
+  const handleClickNumberButton = (value: number) => {
+    const _currNumber = currNumber.current;
+    if (_currNumber?.length === 3) {
```

Comment:
```suggestion
const MAX_NUBER_LENGTH = 3;

if (_currNumber?.length === MAX_NUBER_LENGTH) {
```

매직넘버의 상수화는 리팩토링때보단 코드 작성시 바로 바로 하는게 좋은 것 같아요!